component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentManager.java

```diff
@@ -211,13 +203,16 @@ public void run() {
 
                 @Override
                 public void close() {
+                    log.info("close component manager.");
```

Comment:
- set message to `Closing ComponentManager`
- set level to debug

Reply:
done